index.js

```diff
@@ -1,20 +1,15 @@
 // const passElement = document.getElementById('pass').value;
 const decodeButton = document.querySelector('.btn').onclick = function startDecode() {
 
-        // while (i < 10000) {
-        //     if( i === passElement.value)
-        //     alert( i )
-        //     i++
-        //     break
-        // }
-    let i = 0
-if (i < passElement.value) {i++}
-if (i = passElement.value) alert( i )
+        while (i < 10000) {
+        if( i == passElement.value) {
+            break;
+        } 
+        i = i + 1}
+        alert(i);
```

Comment:
this part should be inside `i == passElement.value` because alert should be showed only if we find a value. In other cases alert should be ('Password did not found) e.t.c